business_requirement/models/business.py

```diff
@@ -265,10 +265,8 @@ def name_search(self, name, args=None, operator='ilike', limit=100):
         return list(set(names) | set(descriptions))[:limit]
 
     @api.multi
-    @api.returns('self', lambda value: value.id)
-    def message_post(self, body='', subject=None, message_type='notification',
-                     subtype=None, parent_id=False, attachments=None,
-                     content_subtype='html', **kwargs):
+    @api.returns('mail.message', lambda value: value.id)
+    def message_post(self, **kwargs):
```

Comment:
Why not putting full signature if you are later putting specific keyword arguments?